libs/ui/src/reboot_from_usb_button.test.tsx

```diff
@@ -34,9 +34,7 @@ test('renders without a USB drive as expected.', async () => {
       </button>
     </div>
   `);
-  await act(
-    async () => await fireEvent.click(screen.getByText('Reboot from USB'))
-  );
+  await act(async () => fireEvent.click(screen.getByText('Reboot from USB')));
```

Comment:
any idea why we need the `await act(async () => ...)` here?

Reply:
No. We may not need it. Will try.

Reply:
That worked, with the subsequent `getByText` changed to `await findByText`.